src/api/routes/get.js

```diff
@@ -40,18 +34,17 @@ router.get('/reviews/', (req, res) => {
     .then(data => getReviews(data))
     .then(result => res.send({
       product_id,
-      page: (page - 1) * count,
+      page, // :(page - 1) * count, - heroku version that doesn't make sense
       count,
       result
     }))
-    .catch(e => console.error(e.stack));
+    .catch(e => res.sendStatus(404));
```

Comment:
Typically, server errors result in a 500 status, not 404. You might want to have a specific check to make sure only one kind of error results in 404, not any error

Reply:
ok thanks.. i used this one because it’s the one heroku responds with, but now that you mention it, I think its specific to invalid product id